test-report/README.md

```diff
@@ -18,8 +18,8 @@ Shows test results in GitHub UI: .NET (xUnit, NUnit, MSTest), Dart, Flutter, Jav
 | `list-suites`   | Limits which test suites are listed. Supported options:<br>  - all<br>  - failed<br>                                                                                                                                                                  | `true`   | `all`         |
 | `list-tests`    | Limits which test cases are listed. Supported options:<br>  - all<br>  - failed<br>  - none<br>                                                                                                                                                       | `true`   | `all`         |
 | `fail-on-error` | Set this action as failed if test report contain any failed test                                                                                                                                                                                      | `true`   | `true`        |
+| `fail-on-empty` | Set this action as failed if no test results are found                                                                                                                                                                                                | `true`   | `false`       |
 | `only-summary`  | Allows you to generate only the summary.<br>If enabled, the report will contain a table listing each test results file and the number of passed,<br>failed, and skipped tests.<br>Detailed listing of test suites and test cases will be skipped.<br> | `false`  | `false`       |
-| `output-to`     | The location to write the report to. Supported options:<br>  - checks<br>  - step-summary<br>                                                                                                                                                         | `false`  | `checks`      |
```

Comment:
This is a breaking change, and as far as I see it's actually used:
- https://github.com/search?q=elastic%2Foblt-actions%2Ftest-report+%28path%3A.github%2Fworkflows+OR+path%3A**%2Faction.yml+OR+path%3A**%2Faction.yaml%29+output-to&type=code